config/environments/development.rb

```diff
@@ -48,12 +48,16 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
-  # Use an evented file watcher to asynchronously detect changes in source code,
+  # Use an evented file watcher to asynchronously detect changes in source code,config/environments/development.rb
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
   #EMAIL
   #EMAIL_FROM = 'email_from@example.com'
-  #EMAIL_TO = ['rmial_to1@example.com', 'email_to2@example.com']
+  #EMAIL_TO = ['email_to1@example.com', 'email_to2@example.com']
+
+  #Password for request
+  PASSWD = 'changelater'
```

Comment:
Could this be removed in favor of using an environment variable (ENV['password']).  When we deploy the application it'll be much nicer if we don't have to modify any code.  This is also true of the email addresses.